src/JRPC-Server/JRPCHTTPServer.class.st

```diff
@@ -36,11 +36,6 @@ JRPCHTTPServer >> initialize [
 				to: [ :request :response | response entity: ( ZnEntity json: ( self handleJSON: request contents ) ) ] )
 ]
 
-{ #category : #accessing }
-JRPCHTTPServer >> port [
```

Comment:
This method is yet never sent but maybe it should be kept no?

Reply:
If you think is useful we can keep it. Anyway there are other server options that can be exposed, like `#debugMode` and `#serverUrl`, probably I will need it in the future